Website color schemes.txt

```diff
@@ -0,0 +1,27 @@
+
+Primary color 1 : #0c1f31
+
+Secondary : #ac9766
+-----------------------------------------

```

Comment:
I think we could use a specified "accent" colour as well, in-case we need it. This could be "white" or #fff;, as you are referencing below. 

- [ ] Specify "white" as your accent colour, or write down another accent colour